garrysmod/gamemodes/terrortown/gamemode/cl_init.lua

```diff
@@ -368,6 +370,9 @@ function CheckIdle()
 
          timer.Simple(0.3, function()
                               RunConsoleCommand("ttt_spectator_mode", 1)
+                               net.Start("TTT_Spectate")
+                                 net.WriteBool(true)
+                               net.SendToServer()

```

Comment:
No. Because then this PR wouldn't make sense.
You can overwrite ttt_spectator_mode. This is the reason why he created this PR.
So if you do this net message in the cvar change callback then this wouldn't get called if you overwrite the convar with alias.